app.py

```diff
@@ -39,13 +73,29 @@ def search():
     query = request.args.get('query', None)
     if query:
         print('query is %s' % query)
-        res = simple_match_search(ES, 'news', query)
-        list_res = res['hits']['hits']
-        for one in list_res:
-            # u6250082 Xuguang Song
-            sum_txt = body_summary(one['_source']['art'])
-            one['_source']['summary'] = ' '.join(sum_txt)
-        return jsonify(list_res)
+
+        in_result = rd.execute_command("cf.check", "test2", hash(query), fingerprint(query))
+        if in_result:
```

Comment:
**[Required]** This comparison will need to be changed to cast `in_result` to an int since it is a string as is. Something like this:

```python
if int(in_result) == 1:
```